lib/recurly.js

```diff
@@ -374,7 +433,8 @@ export class Recurly extends Emitter {
 }
 
 Recurly.prototype.Frame = Frame;
-Recurly.prototype.coupon = require('./recurly/coupon');
+Recurly.prototype.coupon = coupon;
+Recurly.prototype.giftCard = giftcard;
```

Comment:
dup line

Reply:
it's actually a case change. I'll make a note on the `giftcard` line that marks it as deprecated